code/mob/melee_attack_procs.dm

```diff
@@ -65,12 +65,16 @@
 
 	playsound(src.loc, 'sound/impact_sounds/Generic_Shove_1.ogg', 50, 1, -1)
 	if (src == target)
-		var/item = src.get_random_equipped_thing_name()
-		if (item)
-			var/v = pick("tidies","adjusts","brushes off", "flicks a piece of lint off", "tousles", "fixes", "readjusts","fusses with", "sweeps off")
-			src.visible_message("<span style=\"color:blue\">[src] [v] [his_or_her(src)] [item]!</span>")
+		var/obj/stool/S = (locate(/obj/stool) in src.loc)
+		if (S)
+			S.buckle_in(src,src)
```

Comment:
Beds are special and use `buckle_mob()` which checks some stuff and then calls `buckle_in()`. Would recommend refactoring code.